static/js/apps/nl_interface/query_search.tsx

```diff
@@ -20,13 +20,22 @@
 
 import _ from "lodash";
 import React, { useState } from "react";
-import { Button, Container } from "reactstrap";
+import { Button, Container, FormGroup, Input, Label } from "reactstrap";
 
 import { TextSearchBar } from "../../components/text_search_bar";
 
+export const NL_INDEX_SMALL = "small";
```

Comment:
since these constants are used across the NL interface app, let's put them in nl_interface_constants.ts

Reply:
Done.